front/migrations/20240423_backfill_customerio.ts

```diff
@@ -0,0 +1,44 @@
+import * as _ from "lodash";
+
+import { renderUserType } from "@app/lib/api/user";
+import { User } from "@app/lib/models/user";
+import { AmplitudeServerSideTracking } from "@app/lib/tracking/amplitude/server";
+import { CustomerioServerSideTracking } from "@app/lib/tracking/customerio/server";
+import logger from "@app/logger/logger";
+import { makeScript } from "@app/scripts/helpers";
+
+const backfillCustomerIo = async (execute: boolean) => {
```

Comment:
Let make sure that this does not send gazillion emails as we run it :grimacing:

Reply:
Why would it send emails ?

Reply:
This just creates the users on CIO side